actors/miner/src/state.rs

```diff
@@ -64,7 +64,7 @@ pub struct State {
     pub locked_funds: TokenAmount,
 
     /// VestingFunds (Vesting Funds schedule for the miner).
-    pub vesting_funds: Cid,
+    pub vesting_funds: VestingFunds,
```

Comment:
Could make this entire thing optional? Not sure if it's worth it?

Reply:
Basically, we can't _delete_ dead miners but we can _minimize_ them. One part of that is turning fields in the miner actor into "options".

But in this case, the size of an "empty" vesting funds will be 3 bytes, so it's probably not worth it to shrink that to 1 byte.